evennia/typeclasses/models.py

```diff
@@ -733,3 +736,135 @@ def at_rename(self, oldname, newname):
+        HTML anchor.
+        
+        This method is naive and simply returns a path. Securing access to
+        the actual view and limiting who can view this object is the developer's 
+        responsibility.
+        
+        Returns:
+            path (str): URI path to object detail page, if defined.
+            
+        """
+        try: return reverse('%s-detail' % self._meta.verbose_name.lower(), kwargs={'pk': self.pk, 'slug': slugify(self.name)})
+        except: return '#'
+        
+    def web_get_update_url(self):
+        """
+        Returns the URI path for a View that allows users to update this
+        object.
+        
+        ex. Oscar (Character) = '/characters/oscar/1/change/'
+        
+        For this to work, the developer must have defined a named view somewhere
+        in urls.py that follows the format 'modelname-action', so in this case
+        a named view of 'character-update' would be referenced by this method.
+        
+        ex.
+        url(r'characters/(?P<slug>[\w\d\-]+)/(?P<pk>[0-9]+)/change/$', CharUpdateView.as_view(), name='character-update')
+        
+        If no View has been created and defined in urls.py, returns an
+        HTML anchor.
+        
+        This method is naive and simply returns a path. Securing access to
+        the actual view and limiting who can modify objects is the developer's 
+        responsibility.
+        
+        Returns:
+            path (str): URI path to object update page, if defined.
+            
+        """
+        try: return reverse('%s-update' % self._meta.verbose_name.lower(), kwargs={'pk': self.pk, 'slug': slugify(self.name)})
+        except: return '#'
+        
+    def web_get_delete_url(self):
+        """
+        Returns the URI path for a View that allows users to delete this object.
+        
+        ex. Oscar (Character) = '/characters/oscar/1/delete/'
+        
+        For this to work, the developer must have defined a named view somewhere
+        in urls.py that follows the format 'modelname-action', so in this case
+        a named view of 'character-detail' would be referenced by this method.
+
+        ex.
+        url(r'characters/(?P<slug>[\w\d\-]+)/(?P<pk>[0-9]+)/delete/$', CharDeleteView.as_view(), name='character-delete')
+        
+        If no View has been created and defined in urls.py, returns an
+        HTML anchor.
+        
+        This method is naive and simply returns a path. Securing access to
+        the actual view and limiting who can delete this object is the developer's 
+        responsibility.
+        
+        Returns:
+            path (str): URI path to object deletion page, if defined.
+            
+        """
+        try: return reverse('%s-delete' % self._meta.verbose_name.lower(), kwargs={'pk': self.pk, 'slug': slugify(self.name)})
+        except: return '#'
+        
+    # Used by Django Sites/Admin
+    get_absolute_url = web_get_detail_url
```

Comment:
Why is `web_get_detail_url` not named `web_absolute_url` if `get_absolute_url` is what django expects - would not that name (just adding `web_`) be more expected?